application/actions/requests_test.go

```diff
@@ -965,6 +965,67 @@ func (as *ActionSuite) Test_requestsGet() {
 	}
 }
 
+func (as *ActionSuite) Test_requestsCreate() {
+	f := createFixturesForRequestQuery(as)
+
+	destination := api.Location{Description: "location description", Country: "XX"}
+
+	goodRequest := api.RequestCreateInput{
+		Destination:    destination,
+		OrganizationID: f.Organization.UUID,
+		Size:           api.RequestSize(models.RequestSizeSmall),
+		Title:          "request title",
+	}
+	badRequest := api.RequestCreateInput{}
```

Comment:
In addition to the bad and good requests, should we also have a unit test with more than the minimal number of fields, e.g., with origin, photoid, etc. populated in order to verify the code running that logic?

Reply:
I believe the code that handles the data is tested by `Test_convertCreateRequestInput` and `Test_convertRequest`. Do you see something that isn't tested at all?

Reply:
It is! I made the comment before reading through the full PR, but forgot to remove it after seeing that test